docs/downloads/automation-library/share_knowledge.cm

```diff
@@ -0,0 +1,17 @@
+# -*- mode: yaml -*-
+
+manifest:
+  version: 1.0
+
+automations:
+  assign_code_experts:
+    if:
+      - {{ pr.labels | match(term='Share Knowledge') | some }}
+    run:
+      - action: add-reviewers@v1
+        args:
+          reviewers: {{ repo | codeExperts(lt=60) | random }}
```

Comment:
can we add a bottom threshold too (e.g. assign someone with less than 60 but above 30)?

Reply:
What do you think @BenLloydPearson

Reply:
@shadowshot-x I think you can do this: `codeExperts(lt=60, gt=30)`

Reply:
Thank you @vim-zz @yishaibeeri . I was testing this so it took some time. This is working. Great recommendation :D